lib/internal/fs/streams.js

```diff
@@ -350,6 +362,12 @@ function WriteStream(path, options) {
     validateFunction(this[kFs].close, 'options.fs.close');
   }
 
+  this.flush = options.flush ?? false;
+  validateBoolean(this.flush, 'options.flush');
```

Comment:
We don't need to validate if options.flush does not exist since default value is false

Reply:
It's validating that a boolean was actually passed.

Reply:
But it validates it whether or not you pass it. You don't need to validate if it's null or undefined because we assume that the default value is false.